wheatley/main.py

```diff
@@ -53,13 +53,14 @@ def create_row_generator(args):
     return row_gen
 
 
-def create_rhythm(peal_speed, inertia, max_bells_in_dataset, handstroke_gap, use_wait):
+def create_rhythm(peal_speed, inertia, max_bells_in_dataset, handstroke_gap, use_wait, initial_inertia=None):
```

Comment:
Should this be `None` or a number (e..g `1.0`)?

Reply:
It's an optional number - if the parameter is `None`, the Rhythm will make it the same as `--inertia`